charmtools/templates/python/template.py

```diff
@@ -64,7 +65,10 @@ def _template_file(self, config, outfile):
         o = tempfile.NamedTemporaryFile(
             dir=path.dirname(outfile), delete=False)
         os.chmod(o.name, mode)
-        o.write(str(t))
+        st = str(t)
+        if sys.version_info >= (3, ):
```

Comment:
Is this check needed? Do we have to support both python2 and python3?

Reply:
Unfortunately, we do.  charm-tools is included directly via PyPI in some CI systems, though I don't know if `charm create` would actually be used by those.

Note, this was copied from one of the templates that [already included the check](https://github.com/juju/charm-tools/blob/master/charmtools/templates/bash/template.py#L64-L67).